portfolio/src/main/webapp/script.js

```diff
@@ -113,8 +113,16 @@ function initComments() {
       commentsLanguageElement.add(optionElement);
     }
 
+    // If the user didn't choose other language before, set it to 'original'
+    if(!localStorage.commentsLanguageCode)
```

Comment:
What's the type of 'commentsLanguageCode'? Can you really do the boolean operation on it?

Reply:
Solved, I replaced it with a hasOwnAttribute function.

Thanks for pointing out!